res/skins/Deere/deck_visual_row.xml

```diff
@@ -66,6 +66,15 @@
         - if a MIDI device which supports more hotcues than buttons are in the current skin has them activated
         - if you change from a skin which supports more hotcues than buttons are in the current skin (and has them activated)
         -->
+        <Mark>
```

Comment:
I restored to the initial, working state. Personally, i think the graphics should be removed as well for loops and CUE points, but there was no feedback yet in the original bug report https://bugs.launchpad.net/mixxx/+bug/1750210

Reply:
What would you replace them with? A letter?

Reply:
I do think the default rendering suits best - partly because of limitations in the subpixel rendering of the images. 
* For hotcues we allow to name hotcues (for now only in CUE tab of the tracks´ preferences). Adding an image will overwrite the hotcues´ label. 
 So + 1 for me here for default rendering.
* For Loop-In, Loop-Out, and CUE points, i do think using images is not preferable for now, because of errors in the rendering of the images.

  You can see it clearly when you have quantize enabled, and trying to exactly set a CUE image on a beat-grid. Even if your image is only 1px wide, it gets randomly placed left, right, or directly on the beat-grid line. 

   Skins that currently use marker images dodge this by making the vertical line off center, or 2+ pixel wide.